purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -57,6 +62,17 @@ open class PurchasesConfiguration(builder: Builder) {
             this.appUserID = appUserID
         }
 
+        /**
+         * Enable this setting to show a toast with recovery options for users who have had a declined payment
+         * automatically. https://developer.android.com/google/play/billing/subscriptions#in-app-messaging
+         *
+         * If this setting is disabled, you can show the toast by calling
+         * [Purchases.showDeclinedPaymentMessageIfNeeded]
+         */
+        fun showDeclinedPaymentMessagesAutomatically(showDeclinedPaymentMessagesAutomatically: Boolean) = apply {
```

Comment:
not married to the naming

Reply:
I've been thinking about it... From the documentation, this seems to be the only use case for inapp messages right now, but the naming in Google Play seems more generic, so I wonder if we should keep it generic as well and have it be `showInAppMesssagesAutomatically`... Will keep it like this for now since it's clearer what's going to be doing for now, but we might need to rename it in the future if Google adds more functionality.